lib/Models/Catalog/Esri/ArcGisMapServerCatalogItem.ts

```diff
@@ -552,7 +555,13 @@ export default class ArcGisMapServerCatalogItem extends UrlMixin(
         /** Only used "pre-cached" tiles if we aren't requesting any specific layers
          * If the `layersArray` property is specified, we request individual dynamic layers and ignore the fused map cache.
          */
-        usePreCachedTilesIfAvailable: this.layersArray.length == 0,
+        usePreCachedTilesIfAvailable:
+          this.layersArray.length === 0 ||
+          !this.layers ||
+          arraysAreEqual(
```

Comment:
If the arrays are only different in order, should still set it to true.

Reply:
Great point, I have changed it to `setsAreEqual`